pkg/editor/editor.go

```diff
@@ -0,0 +1,57 @@
+package editor
```

Comment:
This is an new helper added in PR 

@craicoverflow it can be used also in other parts of the CLI - for example specifying some complex topic structures and in future for Authz rules.

Reply:
ooh very nice!